### DISCUSSION TITLE
Option to pass to preamble of cached files

### DISCUSSION THREAD

**Discussion by dflvunoooooo** [leo-colisson/robust-externalize#30] [Q&A] [answered]
Is it possible to pass an option to only the cached files, all cached files, but not the main document? Like `runHereAndInPreambleOfCachedFiles` but without the here. To set an option for all presets, but not the main document.

**Comment by tobiasBora** [comment, marked as answer]
Sure, just use
```
\robExtConfigure{
  add to preset={latex}{
    add to preamble={
      % type here any code going to the preamble of cached files only
    }
  }
}
```

**Comment by dflvunoooooo** [comment]
Yes, thank you. I was at first thinking, it would be nice to have a way to config everything outside of `\robExtConfigure`, but it will get very messy if done so. Thanks for your answer.
